src/components/InteractionLayer/index.js

```diff
@@ -545,23 +545,29 @@ class InteractionLayer extends React.Component {
 
       let s = null;
 
-      if (a.start.xval) {
```

Comment:
Do we check elsewhere if a.start is defined?

Reply:
It's required in the PropTypes